### ISSUE TITLE
Build Failure in /src/pyodide-build

### ISSUE THREAD

**Issue by siefkenj** [pyodide/pyodide#5219]
I just made a fresh clone of the `main` branch and am trying to follow the build instructions for the Docker container. I run `make` inside the docker container and it will build the `.wasm` file for Python but not the standard library. I get the following error:

```
make[1]: Entering directory '/src'
Ensuring pyodide-build is installed
pip install -e ./pyodide-build
Defaulting to user installation because normal site-packages is not writeable
Obtaining file:///src/pyodide-build
ERROR: file:///src/pyodide-build does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
make[1]: *** [Makefile:229: pyodide_build] Error 1
```

**Comment by siefkenj** [comment]
Thanks, that fixed it! Was this somewhere in the build instructions, or do the instructions need to be updated?

**Comment by hoodmane** [comment]
I see no mention of it on this page:
https://pyodide.org/en/stable/development/building-from-sources.html
so I think the instructions need to be updated.

**Comment by siefkenj** [comment]
Closed by #5220
